### ISSUE TITLE
extension_conflict_resolution_t11 is failing

### ISSUE THREAD

**Issue by liamappelbe** [dart-lang/sdk#38185]
co19_2/LanguageFeatures/Extension-methods/extension_conflict_resolution_t11 is failing across multiple bots

Culprit seems to be https://dart-review.googlesource.com/c/sdk/+/115210

Example log: https://logs.chromium.org/logs/dart/buildbucket/cr-buildbucket.appspot.com/8903304515198514960/+/steps/test_results/0/logs/unapproved_failing_tests__logs_/0

**Comment by mraleph** [comment]
I think this has been either approved or fixed. So closing this for now.

**Comment by liamappelbe** [comment]
Yeah, I approved it when I filed this bug. This bug is meant to track the work to fix the test. Isn't that the usual procedure?
